rwArray.go

```diff
@@ -59,17 +59,17 @@ func (a *RWArray) Remove(index uint) {
 	a.slots[index].used = false
 }
 
-func (a *RWArray) Get(index uint) interface{} {
+func (a *rwArray) Get(index uint) interface{} {
```

Comment:
So should the only public methods be `Remove()` and `Get()`?

Reply:
I think we should make a decision, to export RWArray as public, as part of the libbpfgo API, with all its methods public; Or have it and all its methods completely private. 

I'm leaning towards lets make everything private, it's not directly relevant to libbpfgo API, just used as an internal implementation detail.